### PR TITLE
Add Mark as read to end of document action

### PR DESCRIPTION
**Pull request by robert00s**

### CODE REVIEW

File: frontend/apps/reader/modules/readerstatus.lua [hunk 1]
```diff
@@ -122,6 +135,12 @@ function ReaderStatus:onEndOfBook()
         end
     elseif settings == "file_browser" then
         self:openFileBrowser()
+    elseif settings == "mark_read" then
+        self:onMarkBook(true)
+        UIManager:show(InfoMessage:new{
+            text = _("You've reached the end of the document.\nCurrent book mark as read."),
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerstatus.lua]:
```suggestion
            text = _("You've reached the end of the document.\nThe current book is marked as read."),
```

**Comment by Frenzie** [comment]
I think it would make more sense to put `Cancel` in the bottom left. Since Mark as read feels related to book status, that'd basically be a simple swap around like this:

* Mark as read/Book status
* Delete file/Next file
* Cancel/File browser

Any thoughts @poire-z @NiLuJe ?

**Comment by poire-z** [comment]
I agree with the layout you propose, seems logical to me too.

**Comment by poire-z** [comment]
I agree with the layout @Frenzie proposes, seems logical to me too.

**Comment by NiLuJe** [comment]
Yep, makes sense!

**Comment by robert00s** [comment]
@Frenzie Thanks. It really make more sense.

**Comment by Frenzie** [comment]
Thanks!

**Pull request merged**